src/blog/2022/04/flowforge-04-released.md

```diff
@@ -0,0 +1,97 @@
+---
+title: FlowForge 0.4 released
+subtitle: Getting ready for Node-RED 3.0
+description: Getting ready for Node-RED 3.0
+date: 2022-04-14 12:00:00.0
+authors: ["sam-machin"]
+---
+
+This release of the FlowForge add a seemingly small but significant new feature.
+
+<!--more-->
+
+With (Node-RED 3.0 fast approaching)[https://nodered.org/about/releases/] we've been making sure we are ready to support this.
+
+### Upgrading Node-RED
+
+The goal of FlowForge is to be the best way to run Node-RED at scale, whether thats many users or many instances. Node-RED is a constantly developing as a platform and therefore part of running Node-RED is also upgrading the version you are running.
+
+With the 0.4 release today we've made that super simple in FlowForge. Last month we introduced the concept of [Project Stacks](https://flowforge.com/docs/user/concepts/#project-stack), and one of the key elements of a Stack was the version of Node-RED in use. Initially this may have seemed fairly basic, when you create a new project you usually want to use the latest version of Node-RED. However what happens when a new version is released and you have an existing project?
```

Comment:
```suggestion
With the 0.4 release today we've made that super simple in FlowForge. Last month we introduced the concept of [Project Stacks](https://flowforge.com/docs/user/concepts/#project-stack). One of the key elements of a Stack was the version of Node-RED in use. Initially this may have seemed fairly basic, when you create a new project you usually want to use the latest version of Node-RED. However what happens when a new version is released and you have an existing project?
```